app/services/catalog/submit_order.rb

```diff
@@ -23,7 +23,7 @@ def process
 
     def submit_order_item(item)
       TopologicalInventory.call do |api_instance|
-        result = api_instance.order_service_plan(item.service_plan_ref, parameters(item))
+        result = api_instance.order_service_offering(item.portfolio_item.service_offering_ref, parameters(item))
```

Comment:
Looking at https://projects.engineering.redhat.com/browse/TPINVTRY-466 which included these PRs:
https://github.com/ManageIQ/topological_inventory-openshift/pull/99/files
https://github.com/ManageIQ/topological_inventory-ansible_tower/pull/24/files

Neither exposes a method called `order_service_offering`. I'm just checking that `order_service_offering` is the exposed method name via the open api generated client.

Reply:
https://github.com/ManageIQ/topological_inventory-api/pull/235, which both of those PRs depended on.

Reply:
Nice. I saw that operationId name. That's what I was looking for.